### PR TITLE
[draft](meta-service) Optmize cloud `commit_txn` large txn

### PR DESCRIPTION
**Pull request by swjtu-zhanglei**
save workspace

0620 save workspace

## Proposed changes

Issue Number: close #xxx

### CODE REVIEW

**Comment by SWJTU-ZhangLei** [comment]
run buildall

File: cloud/src/meta-service/meta_service_txn.cpp [hunk 1]
```diff
@@ -1238,6 +1430,417 @@ void MetaServiceImpl::commit_txn(::google::protobuf::RpcController* controller,
...
 } // end commit_txn
 
+void MetaServiceImpl::commit_txn_v2(
+        ::google::protobuf::RpcController* controller, const CommitTxnRequest* request,
+        CommitTxnResponse* response, ::google::protobuf::Closure* done, MetaServiceCode& code,
+        std::string& msg, std::stringstream& ss, const std::string& instance_id, int64_t db_id,
+        std::vector<std::pair<std::string, doris::RowsetMetaCloudPB>>& tmp_rowsets_meta) {
+    bool need_commit_last_txn = false;
+    TxnErrorCode err = TxnErrorCode::TXN_OK;
+    do {
```
Comment (gavinchou) [inline cloud/src/meta-service/meta_service_txn.cpp:1440]: too large code block

File: cloud/src/meta-service/meta_service.h [hunk 2]
```diff
@@ -275,6 +272,25 @@ class MetaServiceImpl : public cloud::MetaService {
...
                                                               InstanceInfoPB* instance);
 
+private:
+    void commit_txn_with_sub_txn(::google::protobuf::RpcController* controller,
+                                 const CommitTxnRequest* request, CommitTxnResponse* response,
+                                 ::google::protobuf::Closure* done, MetaServiceCode& code,
+                                 std::string& msg, std::stringstream& ss,
+                                 const std::string& instance_id);
+
+    void commit_txn_v1(
```
Comment (gavinchou) [inline cloud/src/meta-service/meta_service.h:282]: suggest naming, 
commit_txn_immeidately()
commit_txn_lazily()

**Comment by SWJTU-ZhangLei** [comment]
run buildall

**Comment by SWJTU-ZhangLei** [comment]
run buildall

**Comment by SWJTU-ZhangLei** [comment]
run buildall

**Comment by SWJTU-ZhangLei** [comment]
run buildall

**Comment by SWJTU-ZhangLei** [comment]
run buildall

**Comment by SWJTU-ZhangLei** [comment]
run buildall

**Pull request closed without merging**